### PR TITLE
Only localIds that are part of the annotation should be considered fo…

### PR DESCRIPTION
**Pull request by adongare**
…r coverage caculation

Pull requests into cqm-execution require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

File: lib/helpers/measure_helpers.js [hunk 1]
```diff
@@ -102,7 +102,37 @@ module.exports = class MeasureHelpers {
         clause.isUnsupported = true;
       }
     }
-    return localIds;
+    // find all localIds in the annotation
+    const allAnnotatedIds = statement ? this.findAnnotationLocalIds(statement.annotation) : [];
+    // filter out local ids that aren't in the annotation, we don't want to account such localIds for coverage
```
Comment (jkotanchik-SB) [inline lib/helpers/measure_helpers.js:107]:
```suggestion
    // filter out local ids that aren't in the annotation, we don't want to include such localIds for coverage.
```

**Review by jkotanchik-SB — APPROVED** [review]

**Comment by adongare** [comment]
Closing this. created ne PR: https://github.com/projecttacoma/cqm-execution/pull/276
